docs/tutorials/standalone/Soil/freezing_front.jl

```diff
@@ -215,13 +215,13 @@ tendency_jacobian! = ClimaLand.make_tendency_jacobian(soil);
 jac_kwargs =
     (; jac_prototype = ImplicitEquationJacobian(Y), Wfact = tendency_jacobian!);
 
-dt = Float64(10)
+dt = Float64(100)
 
 timestepper = CTS.ARS111();
 ode_algo = CTS.IMEXAlgorithm(
     timestepper,
     CTS.NewtonsMethod(
-        max_iters = 3,
+        max_iters = 23,
```

Comment:
why 23? it seems like 20 would be more straightforward, but it doesn't really matter :)

Reply:
I think this was a typo, going back to 3.